src/stores/CurrentGameStore.ts

```diff
@@ -82,4 +82,18 @@ export class CurrentGameStore {
     }
     return yourHands.slice(-1)[0]
   }
+
+  @computed get sortedParticipations(){
```

Comment:
würde ich löschen und in der view sortieren

Reply:
Stehe ein wenig auf dem schlauch wie ich den umbauen soll.